### DISCUSSION TITLE
sentinel 熔断降级规则不支持全局配置

### DISCUSSION THREAD

**Discussion by zaki-liu** [alibaba/Sentinel#1904] [General]
目前sentinel 熔断降级规则并不支持全局资源配置，有什么好的方式可以减少用户配置规则的工作量？大部分情况下各个资源的熔断降级规则应该是一样的，即使有特殊的情况我们也可以做额外配置，而sentinel不支持全局配置。当网关接入sentinel时，这个问题的影响就比较明显了。

**Comment by sczyh30** [comment]
We may discuss it in #66
